inbox/util/html.py

```diff
@@ -17,6 +29,8 @@ def __init__(self):
         self.fed = []
         self.strip_tag_contents_mode = False
 
+        HTMLParser.__init__(self)
```

Comment:
`HTMLParser` is old style class so `super(...)` cannot be used, but this will work both in Python 2 and 3.

The explicit call to base `__init__` is needed because Python 3 introduces some changes in it, and if you don't call `__init__` the parser state will be broken.